Tests/SisyphosTestApp/SisyphosTests/OtherElementTests.swift

```diff
@@ -0,0 +1,23 @@
+import XCTest
+import SwiftUI
+import Sisyphos
+
+
+final class OtherElementTests: XCTestCase {
+    func testOtherElementCaptured() {
```

Comment:
This test case only tests the matching for one case: When the element is matched via identifier. There are currently three cases that we would need to test:
* matching an `Other` element by identifier (the test you added).
* matching an `Other` element by label
* matching an `Other` element by both identifier and label

Reply:
17368d521c55810587a935efa62a8ad72e8c78d3